### ISSUE TITLE
os10_command failing with error about dictionary

### ISSUE THREAD

**Issue by pokepoke81** [ansible-collections/dellemc.os10#43]
##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME

os10_command

##### ANSIBLE VERSION

```paste below
ansible 2.10.1
  config file = ansible.cfg
  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python3.6/dist-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 3.6.9 (default, Jul 17 2020, 12:50:27) [GCC 8.4.0]
```

##### CONFIGURATION

```paste below
DEFAULT_HOST_LIST(ansible.cfg) = ['environments/prod/hosts', 'environments/prod/headquarters_inventory.vmware.yml']
DEFAULT_ROLES_PATH(ansible.cfg) = ['roles']
HOST_KEY_CHECKING(ansible.cfg) = False
INVENTORY_ENABLED(ansible.cfg) = ['vmware_vm_inventory', 'script', 'ini']
```

##### OS / ENVIRONMENT

Ubuntu 18.04

##### STEPS TO REPRODUCE

I'm migrating from the Ansible default dellos10_command

group_vars.yml
```yaml
ansible_connection: network_cli
ansible_network_os: dellemc.os10.os10
ansible_user: admin
ansible_password: XXXXXXX
ansible_become: yes
ansible_become_method: enable
ansible_become_password: XXXXXXXX
```

playbook
```yaml
- name: Backup Dell OS10 Switch 1
  hosts: 10.22.5.101
  collections:
    - dellemc.os10
  gather_facts: no
  tasks:
    - name: Show Version
      os10_command:
        commands: 
          - show version
```

##### EXPECTED RESULTS

Show the version

##### ACTUAL RESULTS

Fails with error

```paste below
The full traceback is:
WARNING: The below traceback may *not* be related to the actual failure.
  File "/tmp/ansible_os10_command_payload_36NueE/ansible_os10_command_payload.zip/ansible/module_utils/basic.py", line 1879, in _handle_elements
    validated_params.append(type_checker(value, **kwargs))
  File "/tmp/ansible_os10_command_payload_36NueE/ansible_os10_command_payload.zip/ansible/module_utils/basic.py", line 1757, in _check_type_dict
    return check_type_dict(value)
  File "/tmp/ansible_os10_command_payload_36NueE/ansible_os10_command_payload.zip/ansible/module_utils/common/validation.py", line 431, in check_type_dict
    raise TypeError("dictionary requested, could not parse JSON or key=value")
[DEPRECATION WARNING]: Distribution Ubuntu 18.04 on host 10.22.5.101 should use
 /usr/bin/python3, but is using /usr/bin/python for backward compatibility with
 prior Ansible releases. A future Ansible release will default to using the 
discovered platform python for this host. See https://docs.ansible.com/ansible/
2.10/reference_appendices/interpreter_discovery.html for more information. This
 feature will be removed in version 2.12. Deprecation warnings can be disabled 
by setting deprecation_warnings=False in ansible.cfg.
fatal: [10.22.5.101]: FAILED! => {
    "ansible_facts": {
        "discovered_interpreter_python": "/usr/bin/python"
    },
    "changed": false,
    "invocation": {
        "module_args": {
            "commands": [
                "show version"
            ],
            "interval": 1,
            "match": "all",
            "retries": 10
        }
    },
    "msg": "Elements value for option commands is of type <type 'str'> and we were unable to convert to dict: dictionary requested, could not parse JSON or key=value"
}
```

**Comment by gunasekaran-t** [comment]
@pokepoke81 , we are working on the fix. Meanwhile you can use it in the below format

```
  tasks:
    - name: show version
      os10_command:
        commands: 
          - command: show version

```

**Comment by woyteck1** [comment]
Will this fix also fix same behavior for os10_config? I was just about to raise it when noticed this issue raised...

**Comment by gunasekaran-t** [comment]
os10_config does not have this issue. Should work fine with the below usage.

```
    - name: configure hostname
      os10_config:
        lines:
          - hostname OS10-1
```

**Comment by gunasekaran-t** [comment]
Fix is available in collection version 1.0.2
